dockerfiles/mlrun-kfp/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 Iguazio
```

Comment:
I think we can take the regular mlrun/mlrun image and install kfp on top of it
we're gonna need this image to run remote workflow
as mlrun/mlrun - used by client pods, dont need kfp

so the image can have much less "things" on it. no need for conda because no need openmpi
we just need mlrun + kfp